config/providers.js

```diff
@@ -1,8 +1,6 @@
 /* eslint-env node */
 'use strict';
 
-const deepFreeze = require('deep-freeze');
```

Comment:
This is the only place where `deep-freeze` is used. So if you're removing it from here, probably it'd make sense to remove the dependency at all. Although, to be honest, I'd prefer to keep it here (and for other configs eventually).